packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -197,9 +205,9 @@ export interface IContextualMenuItem {
   href?: string;
 
   /**
-   * A collection of submenu items
+   * A collection of submenu items, or an IContextualMenuProps object to apply for the sub menu
    */
-  items?: IContextualMenuItem[];
+  items?: IContextualMenuItem[] | IContextualMenuProps;
 
```

Comment:
I can't say I agree. What would the behavior be if a consumer specified both `items` and this separate `submenuProps`? The two are mutually exclusive.